include/envoy/server/hot_restart.h

```diff
@@ -65,6 +65,11 @@ class HotRestart {
   virtual void terminateParent() PURE;
 
   /**
+   * Shutdown admin processing in the current process.
+   */
+  virtual void shutdownAdmin() PURE;
```

Comment:
nit: This doesn't really have anything to do with admin. Can we just call this shutdown() ?